ui/pages/groups/index.js

```diff
@@ -28,8 +28,12 @@ const columns = [
     Cell: ({ value: group }) => {
       return (
         <div className='flex items-center py-1.5'>
-          <div className='flex h-7 w-7 select-none items-center justify-center rounded-md border border-violet-300/40'>
-            <img alt='group icon' src='/groups.svg' className='h-3 w-3' />
+          <div className='flex h-7 w-7 select-none items-center justify-center rounded-md border border-gray-800'>
+            <img
+              alt='group icon'
```

Comment:
Can we use heroicons here?

Also we should use `text-gray-500` or similar as the stroke color since the full white sticks out a bit too much

Reply:
<img width="1246" alt="Screen Shot 2022-08-10 at 12 47 43 AM" src="https://user-images.githubusercontent.com/63033505/183818084-3580b12a-8c17-4102-a60e-7e9a38995e91.png">

so I decided to use `text-gray-400` because it looks too dark for `text-gray-500`